.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,91 @@
+name: Integration tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      platforms:
+        description: 'CSV of Desktop, Android and/or iOS'
+        default: 'Desktop'
+        required: true
+      apis:
```

Comment:
nit: you definitely have more context about this but I noticed its called "apis" here (and in several places in documentation) but python script flag is "--testapps".

Reply:
Yeah, 'testapp' is a carryover for historical reasons, but 'api' seems more intuitive. 

Then again, when triggering, the default value will make it immediately clear what it is, so perhaps it's better to go with that for consistency.

Reply:
I personally have been confused in the past as to what the test app is.  Is it an integration test, a unit test, etc.   I know it's contextualized by the fact that the workflow is integration_test but I think prefer "api" myself.  Just my $0.02.